internal/services/netapp/netapp_volume_resource.go

```diff
@@ -503,6 +514,16 @@ func resourceNetAppVolumeUpdate(d *pluginsdk.ResourceData, meta interface{}) err
 		Properties: &volumes.VolumePatchProperties{},
 	}
 
+	if d.HasChange("zones") {
+		return fmt.Errorf("zone changes are not supported after volume is already created, %s", id)
+	}
+
+	if d.HasChange("storage_quota_in_gb") {
+		shouldUpdate = true
+		storageQuotaInBytes := int64(d.Get("storage_quota_in_gb").(int) * 1073741824)
+		update.Properties.UsageThreshold = utils.Int64(storageQuotaInBytes)
+	}
+
```

Comment:
```suggestion
```
This is duped by accident? Same block is below.

Reply:
Thanks for good catch @favoretti , seems a conflict merge issue. Fixing it.